### PR TITLE
IPv6 network attachements - no dual stack

### PR DESCRIPTION
**Pull request by hjensas**
Add support for IPv6 in network attachement definitions. This does not enable dual-stack support.

**NOTE**:
Dual stack should be possible by utilizing `ipRanges` in the whearabouts ipam config, but attempts to use `ipRanges` fails with an error indicating the wherabouts version in OCP does not have this feature.
See: https://github.com/openstack-k8s-operators/install_yamls/pull/687#issuecomment-1890586935

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -2010,6 +2012,16 @@ ifeq ($(NETWORK_BGP), true)
 netattach: export INTERFACE_BGP_1=${NNCP_BGP_1_INTERFACE}
 netattach: export INTERFACE_BGP_2=${NNCP_BGP_2_INTERFACE}
 endif
+ifeq ($(NETWORK_ISOLATION_IPV4), true)
+netattach: export IPV4_ENABLED=true
+netattach: export CTLPLANE_IPV6_ADDRESS_PREFIX=${NNCP_CTLPLANE_IP_ADDRESS_PREFIX}
```
Comment (abays) [inline Makefile:2017]: Should this be `*_IP_*` instead of `*_IPV6_*`?

Comment (hjensas) [reply Makefile]: oh my, yes it should. Fixed.

**Review by abays — APPROVED** [review]
/lgtm

**Comment by abays** [comment]
recheck

`RETRY_LIMIT`

File: scripts/gen-netatt.sh [hunk 2]
```diff
@@ -39,11 +39,27 @@ if [ -z "${VLAN_STEP}" ]; then
...
 echo INTERFACE ${INTERFACE}
 echo VLAN_START ${VLAN_START}
 echo VLAN_STEP ${VLAN_STEP}
-echo CTLPLANE_IP_ADDRESS_PREFIX ${CTLPLANE_IP_ADDRESS_PREFIX}
+if [ -n "$IPV4_ENABLED" ]; then
+    echo CTLPLANE_IP_ADDRESS_PREFIX ${CTLPLANE_IP_ADDRESS_PREFIX}
+    echo CTLPLANE_IP_ADDRESS_SUFFIX ${CTLPLANE_IP_ADDRESS_SUFFIX}
+fi
+if [ -n "$IPV6_ENABLED" ]; then
+    echo echo CTLPLANE_IPV6_ADDRESS_PREFIX ${CTLPLANE_IPV6_ADDRESS_PREFIX}
```
Comment (rabi) [inline scripts/gen-netatt.sh:60]: echo echo ?

Comment (hjensas) [reply scripts/gen-netatt.sh]: Fixed.

**Comment by abays** [comment]
recheck

Compute kernel panic:

```
[   11.739532] ---[ end Kernel panic - not syncing: Jitter RNG permanent health test failure ]---
```

**Pull request merged**